app/l10n.py

```diff
@@ -0,0 +1,12 @@
+from flask import request
```

Comment:
I decided to create a new `l10n` module because all the l10n stuff in `app/factory.py` was starting to get really crowded, and I didn't want to add to the already big list of imports in that module... In a future PR I'd like to migrate more of the localization code into `l10n.init_app()`.